src/components/layers-manager/CustomLayers.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Input, Textarea } from '../form/Form';
+import { injectIntl } from 'react-intl';
+
+import withModal from '../ui/withModal';
+import Walkthrough from '../ui/Walkthrough';
+import Icon from '../ui/Icon';
+
+const WalkthroughModal = withModal(Walkthrough);
+
+const WALKTHROUGH_TEXTS = [
+  {
+    content: 'customLayers.step1'
+  },
+  {
+    content: 'customLayers.step2'
+  },
+  {
+    content: 'customLayers.step3',
+    childContent: 'customLayers.step3.1'
+  },
+  {
+    content: 'customLayers.step4'
+
+  }
+];
+
+class CustomLayers extends Component {
+
+  static propTypes = {
+    form: PropTypes.object.isRequired,
+    onInputChange: PropTypes.func.isRequired,
+    intl: PropTypes.object.isRequired
+  };
+
+  state = {
```

Comment:
I prefer this way, it avoid having to declare a constructor and the need to call `super`.